pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -239,9 +239,10 @@ def make_package(args):
     assets_dir = "src/main/assets"
 
     # Delete the old assets.
-    try_unlink(join(assets_dir, 'public.mp3'))
-    try_unlink(join(assets_dir, 'private.mp3'))
+    if os.path.exists(assets_dir):
+        shutil.rmtree(assets_dir)
     ensure_dir(assets_dir)
+    open(os.path.join(assets_dir, ".gitkeep"), 'a').close()
```

Comment:
Is this needed? Even if you're adding the output to git, it seems like this stuff should be handled at that later stage (although it's acceptable if there's some good reason for it).

Reply:
I asked about this on Discord. As it stands now, it's fine if the assets directory is completely deleted and recreated each time, but the behaviour *used* to be that the assets directory was kept so I tried to be consistent with that.

Reply:
Got it. I think it should be fine to ditch it and let the directory be deleted if you want, but if leaving this in could you add a comment noting this reasoning?